aries-site/src/pages/components/maskedinput.js

```diff
@@ -22,11 +34,178 @@ const MaskedInput = () => {
           <Example
             code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/MaskedInputExample/MaskedInputExample.js"
             docs="https://v2.grommet.io/maskedinput?theme=hpe#props"
+            figma="https://www.figma.com/file/fO285pXUqN9pVZ0xpRoKjL/HPE-Masked-Input-Component?node-id=1%3A89"
           >
-            <MaskedInputExample />
+            <MaskedPhoneExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            MaskedInput place syntax restrictions on the text that can be
```

Comment:
This paragraph does a good job explaining / defining **what** MaskedInput is. I am wondering if we can add a **why** statement at the beginning of the paragraph which sells me on why I should reach for MaskedInput and how it improves the User Experience (e.g. quick, efficient, error free, minimizes clicks, ability to reduce multiple selects into a single input, progressively reveals info right as I need it, etc. ...).

Reply:
I like that, let me add something in.